CHANGELOG.md

```diff
@@ -32,6 +32,10 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#4041](https://github.com/realm/SwiftLint/issues/4041)
 
+* Make `comma_inheritance` an opt-in rule.  
+  [Steve Madsen](https://github.com/sjmadsen)
+  [#4027](https://github.com/realm/SwiftLint/issues/4027)
```

Comment:
This should rather be mentioned in the "Breaking" section since users need to change their configuration if they want to keep the rule enabled.

Reply:
Moved.